src/db/getOneRecord.spec.js

```diff
@@ -0,0 +1,33 @@
+import sinon from 'sinon';
+import { getOneRecord } from './getOneRecord';
+import AWS from 'aws-sdk';
+import { config } from '../config/dynamoConfig';
+
+describe('db - getOneRecord()', () => {
+    const sandbox = sinon.createSandbox();
+    let pk, name;
+    const promiseStub = sandbox.stub();
+    const clientStub = {
+        get: sandbox.stub().returns({
+            promise: promiseStub,
+        }),
+    };
+    before(() => {
+        sandbox.stub(AWS.DynamoDB, 'DocumentClient').returns(clientStub);
+    });
+    after(() => {
```

Comment:
```suggestion
    afterEach(() => {
```
in case there has to be more than one test